homeassistant/const.py

```diff
@@ -228,6 +228,10 @@
 VOLUME_GALLONS = 'gal'  # type: str
 VOLUME_FLUID_OUNCE = 'fl. oz.'  # type: str
 
+# Bandwidth units
+BANDWIDTH_MEGABITS_SECONDS = 'Mbps'  # type: str
+BANDWIDTH_GIGABITS_SECONDS = 'Gbps'  # type: str
```

Comment:
Because if only one platform need something we don't add it to `const.py`. Then the constants should only be defined in the platform itself. Well, it looks like that we are not consistent with those units anyway. 

I was talking about the form for the units...